integration/use-cases/upgrade.js

```diff
@@ -1,14 +1,12 @@
 test("Upgrades an application", async () => {
-  await page.goto(getUrl("/#/login"));
+  await page.goto(getUrl("/#/c/default/ns/default/catalog"));
```

Comment:
I thought about changing this as well but the problem is that now we are not covering the case of an upgrade in the kubeapps namespace :man_shrugging: In any case, since both cases are covered with unit tests is fine.

Reply:
Yeah - we could easily add another, but I think testing the deploy and upgrade of a global chart in a namespace other than the `kubeapps` one is by far more relevant for the integration tests.